project.md

```diff
@@ -11,41 +11,96 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+People often brainstorm ideas by writing down whatever comes to mind and organizing them later. This can be students trying to find ideas and sources for an essay, travelers planning a trip to some destination, or an all-purpose note sheet for random thoughts and to-dos. However, the result of brainstorming is often hard to parse because of information overload. Nonetheless, there is generally some existing organization within the brainstormed document and a desired organization of these ideas post-brainstorming. An example would be turning potential places to visit into a formal itinerary for the traveler's case.
```

Comment:
I am a huge fan of this project idea because it is something I would actually use. When I started to brainstorm ideas for projects in HW3 I tried to think of areas in my life that technology seemed to fall short. One of the main topics that I found myself stuck thinking about was how methods of idea organization could be improved-- this ultimately led me to choose my project topic, calendar creation. I like your project topic so much because it is a tool I could actually see myself using as a student. From a high level, this the greatest similarity I see between our two projects.